### PR TITLE
Added title block instruction.

### PR DESCRIPTION
**Pull request by hansjovis**
## Summary

This PR can be summarized in the following changelog entry:

* [schema-blocks] Adds a `Title` block instruction.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Companion PR of https://github.com/Yoast/wordpress-seo-premium/pull/3267. Please follow its test instructions.
	* Note: You need to test this only in Premium.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes
* [ ] This PR changes the UI in the plugin. I have added the 'UI change' label to this PR.

## Quality assurance

* [ ] I have tested this code to the best of my abilities
* [x] I have added unittests to verify the code works as intended

Fixes #

### CODE REVIEW

File: packages/schema-blocks/src/instructions/blocks/Title.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+		if ( hasSameTitle ) {
+			return new BlockValidationResult(
+				blockInstance.clientId,
+				blockInstance.name,
+				BlockValidation.Invalid,
+				BlockPresence.Recommended,
+				sprintf(
+					/* Translators: %s expands to the block's name. */
+					__( "Post title and %s are the same.", "wordpress-seo-premium" ),
```
Comment (andizer) [inline packages/schema-blocks/src/instructions/blocks/Title.tsx:55]:
```suggestion
					__( "Post title and %s are the same.", "yoast-schema-blocks" ),
```

**Review by andizer — APPROVED** [review]
CR okay. One minor thing that I can commit myself/

File: packages/schema-blocks/src/instructions/blocks/Title.tsx (hunk 1 above)
Comment (hansjovis) [reply packages/schema-blocks/src/instructions/blocks/Title.tsx]: Thanks! I did not notice that when moving this change from Premium to the schema blocks package...

**Review by andizer — APPROVED** [review]
CR 👍

**Pull request merged**